libs/ui-watt/src/lib/components/input/timepicker/watt-timepicker.module.spec.ts

```diff
@@ -171,6 +173,15 @@ describe(WattTimepickerModule.name, () => {
       expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
     });
 
+    it('disables slider button', async () => {
+      await setup({ template, disabled: true });
+      const sliderToggle = screen.queryByRole('button') as HTMLButtonElement;
+
+      userEvent.click(sliderToggle);
+
+      expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
```

Comment:
question: what do you think about using the `[.toBeDisabled`](https://github.com/testing-library/jest-dom#tobedisabled) matcher on the button in this case? The test wouldn't know what would have happen had the button been clicked.

Reply:
Good thinking, added an extra `expect` to assert for disabled state.